sheetScript.gs

```diff
@@ -520,19 +490,16 @@ function insideUpdate(){
   updatePlaylists(sheet);
 }
```

Comment:
Function can be removed as per https://github.com/Elijas/auto-youtube-subscription-playlist-2/pull/75#discussion_r417007381